src/scss/patterns/_offset-grid.scss

```diff
@@ -49,14 +49,9 @@ $break: 'wide-break';
 .offset-media {
   --offset-media-padding: 0;
   display: block;
-  padding: var(--offset-media-padding);
 
-  &:nth-child(odd) {
-    padding: var(--offset-media-padding);
-  }
-
-  &:nth-child(even) {
-    padding: var(--offset-media-padding);
+  & > * {
+    --offset-media-padding: 0;
   }
```

Comment:
It looks to me like the outer setting (line 50) would cover it, since variables inherit - and this shouldn't be needed.

Reply:
fixed